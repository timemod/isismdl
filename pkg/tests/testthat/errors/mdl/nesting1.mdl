? test nested sum / del and user function
? nested user function allowed in isis version starting from ...

function f1(x) = sum(j = -2, -1 : x(j));
function f2(x) = del(2 : x);;

ident eq1 x1 = sum(k =  -1, 0: f1(x));
ident eq2 x2 = del(2: f1(x));
ident eq3 x3 = sum(k =  -1, 0: f2(x));
ident eq4 x4 = del(2: f2(x));



