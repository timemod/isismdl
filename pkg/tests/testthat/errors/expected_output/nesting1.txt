Use of function f1 implies a nested sum
Line: 7 of file mdl/nesting1.mdl
ident eq1 x1 = sum(k =  -1, 0: f1(x));
                                   ^

Use of function f2 implies a nested del
Line: 10 of file mdl/nesting1.mdl
ident eq4 x4 = del(2: f2(x));
                          ^

